### ISSUE TITLE
Code example of timetracking 

### ISSUE THREAD

**Issue by Konafets** [daaru00/obsidian-jira-issue#1]
I installed and configured both plugins. I am able to download an JIRA issue and starting a timer but how can I combine both?

**Comment by markt2000** [comment]
Yes agreed.  These jira and time-tracker plugins look ideal for my needs but I can't figure out how to get the timer controls to show at the bottom of the jira widget, as in the screenshot.  Any help greatly appreciated, thanks.
